### ISSUE TITLE
@Myprr Getestet auf Chrome: Registrierung funktioniert, wie bei CRM, nun einwandfrei...

### ISSUE THREAD

**Issue by JulianeOrt** [imbitdhbwma/brillianplatform#91]
@Myprr Getestet auf Chrome: Registrierung funktioniert, wie bei CRM, nun einwandfrei...
Allerdings bekomme ich am Ende kein Zertifikat per Mail zu gesendet...
![image](https://user-images.githubusercontent.com/56722211/69743774-4e26d100-113f-11ea-85f2-07dfb6190363.png)

Das ist doch das Ende, oder übersehe ich etwas?

_Originally posted by @StevenHerrenknecht in https://github.com/imbitdhbwma/brillianplatform/issues/44#issuecomment-559171183_

**Comment by JulianeOrt** [comment]
Wie bei CRM sollten bei ICM auch am Ende Zertifikate versendet werden.

Bisher steht nur "Certificate and badge only available for designated groups". Gibt es im ICM Programm schon eine Funktion dahinter für bestimmte Studenten, oder muss das noch neu eingefügt werden?
@ErikGarstka bitte im Technik Team in Sprint 4 bearbeiten!!

**Comment by Feederhigh5** [comment]
Nein, der Spieler (Student) muss zu einer Gruppe von einem Dozenten gehören. Der Dozent kann dann Zertifkate aktivieren und versenden lassen. @StevenHerrenknecht welche E-Mail Adresse hast du verwendet, dann kann man dich in einen Kurs von einem admin hinzufügen, sodass du dann auch testen kannst, ob du ein Zertifikat bekommst.

**Comment by StevenHerrenknecht** [comment]
Hab s.herrenknecht@t-online.de verwendet.

**Comment by StevenHerrenknecht** [comment]
@Feederhigh5 Getestet auf Prod, Zertifikat wurde wie gewünscht versendet!

**Comment by Feederhigh5** [comment]
Nein, habe als Dozent die Zertifkate nochmal an alle senden lassen, daher 2.
Kannst du @StevenHerrenknecht auch die anderen Projekte/Länder testen (inkl. Zertifikat)
